test/GPv2Encoding.test.ts

```diff
@@ -15,16 +21,41 @@ function fillUint(bits: number, byte: number): BigNumber {
 describe("GPv2Encoding", () => {
   const [, ...traders] = waffle.provider.getWallets();
 
+  const testDomain = {};
```

Comment:
According to the standard the domain should have at least one field ;) https://eips.ethereum.org/EIPS/eip-712#definition-of-domainseparator (not sure if it is relevant for the test)

Reply:
Oops I missed that! Ethers.js's EIP-712 implementation happily accepts an empty domain, but I think its better to be correct in this case (even if it is just for testing) in case in the future Ethers.js stops accepting empty domains and breaks tests on a version bump.